eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -111,7 +104,30 @@ steps:
         # Replace windows backslash paths, as Azure Pipelines default directories are sometimes formatted like 'D:\a\1\s'
         $repositories = '${{ convertToJson(parameters.Repositories) }}' -replace '\\', '/' | ConvertFrom-Json -AsHashtable
         foreach ($repo in $Repositories) {
-          SparseCheckout $paths $repo
+          $dir = $repo.WorkingDirectory
+          if (!$dir) {
+            $dir = "./$($repo.Name)"
+          }
+          New-Item $dir -ItemType Directory -Force
+          Push-Location $dir
+
+          try {
+            # Enable global override if there are sparse checkout issues
+            if ('$(DisableSparseCheckout)' -ne 'true') {
```

Comment:
Should we use the usual Skip naming here instead of disable? So SkipSparseCheckout?

Reply:
Perhaps both `Disable` and `Skip` aren't descriptive enough, as they only specify not doing something, as opposed to replacing sparse with a full clone.

Some other ideas:

- OverrideSparseCheckout
- UseFullClone
- CheckoutMode == `sparse|full|clone`

Reply:
I think Override doesn't give any better details of what is actually happening. Maybe CheckoutMode but right now I think there is only 2 options I'm not sure what would be different between full and clone in your options there. 

I don't have a strong feeling but I think Disable or Skip (slightly favoring skip for consistency) would still be the most descriptive of what we are trying to do which is to not run the sparse part of the checkout.

Reply:
yeah re: full vs. clone, I figured we would pick one or the other to mean "full clone." Let's go with skip for now.